portal/oauth/README.md

```diff
@@ -1,37 +1,39 @@
 # OAuth
 
-Use OAuth2 to authenticate with ArcGIS Online (or your own portal) to access secured resources.
+Authenticate with ArcGIS Online (or your own portal) using OAuth2 to access secured resources (such as private web maps or layers). Accessing secured items requires logging in to the portal that hosts them (an ArcGIS Online account, for example).
```

Comment:
This description is pretty long. I think it's unusual that they are more than one sentence?
Should we perhaps move the second sentence to an 'additional information' section?

Reply:
good spot, iOS have already logged a re-wording suggestion for this in the common samples "gotchas" so I'll go with this for consistency.